src/ReflectionMagic/ReflectionMagic.csproj

```diff
@@ -3,8 +3,7 @@
   <PropertyGroup>
     <AssemblyName>ReflectionMagic</AssemblyName>
     <VersionPrefix>4.1.0</VersionPrefix>
-    <TargetFrameworks>net45;netstandard1.5;netstandard2.0</TargetFrameworks>
-    
+    <TargetFrameworks>net6.0;net462;netstandard2.0</TargetFrameworks>
```

Comment:
On my default clean install, I don't have 4.6.2:
    
    MSB3644	The reference assemblies for .NETFramework,Version=v4.6.2 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application...

Is there a downside to going to 4.8? Maybe the downside is to accidentally use 4.8 things, hence making it unusable in 4.6.*?

Bear with me, my .net skills are very rusty!

Reply:
Following Microsoft practices, most NuGet packages with support for .NET Framework are currently targeting .NET Framework 4.6.2 (which will be [supported until Jan 12, 2027][1]). Since this project does not take advantage of APIs introduced in .NET Framework 4.8, I'd stick with supporting .NET Framework 4.6.2.

The error about reference assemblies not found is weird. I have added a `global.json` file that will force using at least the .NET 6.0.400 SDK, where [references assemblies are automatically downloaded][2]. This automatic download behaviour exists since the .NET 5 SDK.

What does `dotnet --info` say when run inside the ReflectionMagic directory?

[1]: https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework
[2]: https://github.com/dotnet/sdk/pull/10981

Reply:
Note that the error above is when building from VS. If I run `dotnet build` from cmd line, I get:

```
global.json file: D:\repos\ReflectionMagic\global.json

Installed SDKs:
7.0.201 [C:\Program Files\dotnet\sdk]

Install the [6.0.400] .NET SDK or update [D:\repos\ReflectionMagic\global.json] to match an installed SDK.
```

Reply:
That's due to the global.json specifying that it requires that specific SDK version. I don't think we really need it, but it does make things more reproducible on the build server side.

Reply:
I installed dotnet core 6.0, and that allowed building using `dotnet build`. It still fails the same from VS. I suspect I could fix it with the dev pack.

I'm a bit confused about the build system from VS vs command line. Are they two totally different things, or did they unify them? A few years ago (last time I used .net), I think a project had to be either .NET Core or .NET full framework, but not both.

Reply:
Installing the dev pack fixes the VS build, so everything works now.

@jvandertil other than the required SDK version (which we can discuss further), are you good with this change?

Reply:
Yes, LGTM.